src/pybind/mgr/devicehealth/module.py

```diff
@@ -0,0 +1,249 @@
+
+"""
+Device health monitoring
+"""
+
+import errno
+import json
+from mgr_module import MgrModule, CommandResult
+import rados
+from threading import Event
+from datetime import datetime, timedelta, date, time
+
+TIME_FORMAT = '%Y%m%d-%H%M%S'
+
+class Module(MgrModule):
+    OPTIONS = [
+        { 'name': 'active' },
+        { 'name': 'scrape_frequency' },
+        { 'name': 'pool_name' },
+        { 'name': 'retention_period' },
+    ]
+    DEFAULTS = {
+        'active': True,
+        'scrape_frequency': '86400',
+        'retention_period': '86400*14',
```

Comment:
when casting to `int` in:
```
def put_device_metrics(self, ioctx, devid, data):
    old_key = datetime.now() - timedelta(
        seconds=int(self.retention_period))
```
we get a runtime error :
`ValueError: invalid literal for int() with base 10: '86400*14'`